src/ui/camera.js

```diff
@@ -31,6 +31,11 @@ import {Aabb, Frustum} from '../util/primitives.js';
 import type {PaddingOptions} from '../geo/edge_insets.js';
 import type {Vec3} from 'gl-matrix';
 
+/**
+ * A helper type: converts all Object type values to non-maybe types.
+ */
+type Required<T> = $ObjMap<T, <V>(v: V) => $NonMaybeType<V>>;
```

Comment:
I'm not sure if this is the right place for this helper type, but I think it's okay to have it here until we need it somewhere else.